src/main/java/com/vmware/gemfire/testcontainers/Credentials.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) VMware, Inc. 2024. All rights reserved.
+ */
+
+package com.vmware.gemfire.testcontainers;
+
+/**
+ * Simple container for credentials
+ */
+public interface Credentials {
+
+  /**
+   * Reference field indicating no credentials are specified
+   */
+  Credentials NONE = of(null, null);
+
+  /**
+   * Create a Credentials instqnce with the given username and password
+   */
+  static Credentials of(final String username, final String password) {
```

Comment:
Do we need to care about `token` rather than just `username/password`

Reply:
@jdeppe-pivotal do we want to support tokens, or maybe leave that for another release?